ibis/pandas/udf.py

```diff
@@ -249,8 +275,14 @@ def aggregator(first, *rest, **kwargs):
                 # TODO: might be inefficient to do this on every call
                 return func(first, *map(next, rest))
 
-            result = aggcontext.agg(args[0], aggregator, *iters, **kwargs)
-
-        return result
+            return aggcontext.agg(args[0], aggregator, *iters, **kwargs)
+        elif isinstance(aggcontext, Window):
+            # We must be aggregating over a bounded window, so we need
+            # aggcontext.agg to handle the rolling/expanding window logic.
+            return aggcontext.agg(args[0], func, *args, **kwargs)
```

Comment:
Can you look into if you can pass *args[1:]

Reply:
Done!

I had to make a change to `window_agg_udf` to make the `Window` `AggregationContext` handle this properly